### PR TITLE
MAT-9547 Updating Transfer dialog for Admin user

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-9546](https://jira.cms.gov/browse/MAT-9546)
(Optional) Related Tickets:

### Summary
Admin UI.

1. No warning text. Updated info text
2. Owner name is displayed in the table.

image

Regalar User UI

1. Mostly remains same except the number of libraries selected will be displayed.

image

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/cqlLibraryLanding/cqlLibraryListActionCenter/transferAction/TransferAction.tsx [hunk 1]
```diff
@@ -36,7 +36,8 @@ export default function TransferAction(props: PropTypes) {
   const [disableTransferBtn, setDisableTransferBtn] = useState(true);
   const [tooltipMessage, setTooltipMessage] = useState(NOTHING_SELECTED);
   const featureFlags = useFeatureFlags();
-  const userRoles: UserRoles = useUserRoles();
+  let userRoles: UserRoles = useUserRoles();
+  userRoles.isAdmin = true; // Temporary override until admin role is implemented in Okta
```
Comment (etan-sb) [inline src/components/cqlLibraryLanding/cqlLibraryListActionCenter/transferAction/TransferAction.tsx]: Do we have an upcoming story to implement this?

File: src/__mocks__/@madie/madie-util.tsx [hunk 2]
```diff
@@ -64,3 +64,23 @@ export function useOnClickOutside(ref, handler) {
...
+  isAdmin: false,
+});
+
+// Mock userRolesStore
+export const userRolesStore = {
+  getState: () => ({ roles: [], isAdmin: false }),
+  updateUserRoles: () => {},
+  subscribe: () => ({ unsubscribe: () => {} }),
+};
+
```
Comment (etan-sb) [inline src/__mocks__/@madie/madie-util.tsx]: Looks like `prettier` failed the build

File: src/components/cqlLibraryLanding/cqlLibraryListActionCenter/transferAction/TransferAction.tsx (hunk 1 above)
Comment (RohitKandimalla) [reply src/components/cqlLibraryLanding/cqlLibraryListActionCenter/transferAction/TransferAction.tsx]: that is debug thingy, Will revert it

File: src/__mocks__/@madie/madie-util.tsx (hunk 2 above)
Comment (RohitKandimalla) [reply src/__mocks__/@madie/madie-util.tsx]: Will check that

File: src/components/cqlLibraryLanding/cqlLibraryListActionCenter/transferAction/TransferAction.tsx (hunk 1 above)
Comment (RohitKandimalla) [reply src/components/cqlLibraryLanding/cqlLibraryListActionCenter/transferAction/TransferAction.tsx]: Done

File: src/__mocks__/@madie/madie-util.tsx (hunk 2 above)
Comment (RohitKandimalla) [reply src/__mocks__/@madie/madie-util.tsx]: Fixed

**Review by etan-sb — APPROVED** [review]
